.github/workflows/danger-prs.yml

```diff
@@ -0,0 +1,19 @@
+name: iOS Code Sanity Checks
+on:
+  pull_request:
+    # The restrictions on the types of pull_request event that can trigger this
+    # workflow is intentional. We only want to run these sanity checks when the
+    # PR is opened or reopened, or its branch receives a push.
+    types: [opened, reopened, synchronize]
```

Comment:
For example, here you can see that after a push only the sanity checks run.

![screencapture-github-mokagio-WordPress-iOS-pull-6-2020-02-11-14_40_12](https://user-images.githubusercontent.com/1218433/74209993-1b23c080-4cde-11ea-9a2b-a564ca640ee6.png)